lib/charms/opensearch/v0/helper_cluster.py

```diff
@@ -65,14 +65,14 @@ def suggest_roles(nodes: List[Node]) -> List[str]:
         return ["data"]
 
     @staticmethod
-    def is_cluster_bootstrapped(nodes: List[Node]) -> bool:
+    def remaining_nodes_for_bootstrap(nodes: List[Node]) -> bool:
         """Check if cluster is bootstrapped. 2 cm + 1 voting only nodes created."""
         nodes_count = ClusterTopology.nodes_count_by_role(nodes)
 
-        cms_ok = nodes_count.get("cluster_manager", 0) == 2
-        voting_only_ok = nodes_count.get("voting_only", 0) > 0
+        cms = 2 - nodes_count.get("cluster_manager", 0) == 2
+        voting_only = 1 - nodes_count.get("voting_only", 0) > 0
```

Comment:
No, but from OpenSearch's guidelines, for a cluster to be considered "Bootstrapped", it mush have "2 cluster managers" + "1 voting_only". And this method returns the remaining number of nodes to reach the "bootstrapped state". 
But there is a bug there, the checks must be dropped, since the return is an int.